platform/linux/supercollider.xml

```diff
@@ -3,8 +3,10 @@
   <mime-type type="text/x-sc">
     <sub-class-of type="text/plain"/>
     <comment xml:lang="en">SuperCollider source code</comment>
+    <comment xml:lang="de">SuperCollider Quelltext</comment>
```

Comment:
OMG, I misread... Didn't see it was entered as German translation. No, no, correct of course. Please ignore the noise. It build without problems on Ubuntu studio, I am just checking that it looks okay, btw.